src/features/AutoCompletion/model/extractActivityidsToSubmitByParams.ts

```diff
@@ -0,0 +1,33 @@
+type ActivityIdsExtractorParams = {
+  isFlow: boolean;
+  currentActivityId: string;
+  flowActivityIds: string[] | null;
+};
+
+export const extractActivityIdsToSubmitByParams = (
+  params: ActivityIdsExtractorParams,
+): string[] => {
+  if (!params.isFlow) {
+    return [params.currentActivityId];
+  }
+
+  if (!params.flowActivityIds) {
+    throw new Error(
+      '[CompletionContructService:extractActivityIdsToSubmitByParams] Flow activity ids are not defined',
```

Comment:
Please change the error text (CompletionContructService mentioned)

Reply:
Done